engine/runtime/src/main/scala/org/enso/interpreter/instrument/NotificationHandler.scala

```diff
@@ -115,14 +115,21 @@ object NotificationHandler {
       libraryName: LibraryName,
       libraryVersion: LibraryVersion,
       location: Path
-    ): Unit = sendMessage(
-      Api.LibraryLoaded(
-        namespace = libraryName.namespace,
-        name      = libraryName.name,
-        version   = libraryVersion.toString,
-        location  = location.toFile
+    ): Unit = {
+      sendMessage(
+        Api.LibraryLoaded(
+          namespace = libraryName.namespace,
+          name      = libraryName.name,
+          version   = libraryVersion.toString,
+          location  = location.toFile
+        )
       )
```

Comment:
I thought that the `Api.LibraryLoaded` should be enhanced with optional list of suggestions. However it looks like the proposal is to send a message to self and deserialize the suggestions later. Assuming the deserialization is instant - is there any benefit of sending the suggestions in a separate message?